doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,650 @@
+   * :ref:`ug_zigbee_other_ecosystems` - New page.
+   * :ref:`ug_zigbee_tools` - Updated with new content and structure.
+
+* Documentation updates in HomeKit and Find My private repositories.
+
+Applications and samples
+------------------------
+
+* nRF9160:
+
+   * :ref:`asset_tracker` - Added sections on using nRF Cloud A-GPS or P-GPS, and on using nRF Cloud FOTA.
+   * :ref:`asset_tracker_v2`:
+
+      * Added a table showing cloud services and the corresponding cloud-side instances.
+      * Extended the documentation to include Azure IoT Hub and nRF Cloud support.
+
+   * :ref:`serial_lte_modem` - Added links to AT command reference guides.
+   * :ref:`agps_sample` - Added a section on using nRF Cloud A-GPS or P-GPS.
+   * :ref:`aws_fota_sample` - Changes in the sample configuration section.
+   * :ref:`fmfu_smp_svr_sample` - Updated the Building and running section.
+   * :ref:`gps_with_supl_support_sample` - Changes in the Overview to reflect the introduction of GNSS interface.
+   * :ref:`lwm2m_client` - Added information about additional configurations.
+
+* nRF5340:
+
+   * :ref:`nc_bootloader` - Reworked the Overview and Building and Running sections.
+
+* Bootloader:
+
+   * :ref:`bootloader` - Sample renamed to |NSIB| and content changed to a large extent.
+
+* Edge Impulse
+
+   * Edge Impulse samples are now in a separate :ref:`edge_impulse_samples` section.
+
+* TF-M
+
+   * :ref:`tfm_hello_world` - Updated the expected sample output and added a reference to the :ref:`lib_tfm_ioctl_api` library, which the sample now uses.
+
+* Thread
+
+   * :ref:`ot_cli_sample` - Updated the procedure for testing Thread 1.2 features.
+   * :ref:`coap_client_sample` - Added information on Device Firmware Upgrade extension and its testing procedure.
+
+* Zigbee
+
+   * Added links to ZBOSS API documentation in :ref:`zigbee_light_bulb_sample`, :ref:`zigbee_network_coordinator_sample`, and :ref:`zigbee_light_switch_sample` samples.
+   * :ref:`zigbee_ncp_sample` - Updated the nRF5 SDK Bootloader section.
+
+Libraries and drivers
+---------------------
+
+* :ref:`bt_mesh` - This overview now points to specifications for general Bluetooth Mesh information.
+* :ref:`caf_overview` - Added a major description of the library with extensive sections describing all of its aspects.
+* :ref:`lte_lc_readme` - Added multiple new sections in this library documentation.
+* :ref:`mpsl_assert` - New library documentation.
+* :ref:`lib_nrf_cloud` - Added sections on Configuration options for device ID and on Firmware over-the-air (FOTA) updates.
+
+nrfxlib
+-------
+
+* :ref:`nrf_802154` - Full documentation of the radio driver is now available as part of nrfxlib and replaces the previous nRF 802.15.4 Service Layer documentation.
+* :ref:`nrf_security_readme` - Added section on Building with TF-M.
+* Added :ref:`zboss_certification`.
+
+Known issues
+************
+
+See `known issues for nRF Connect SDK v1.6.0`_ for the list of issues valid for this release.
```

Comment:
Propose: move known issues above "change log"

```suggestion
Known issues
************

See `known issues for nRF Connect SDK v1.6.0`_ for the list of issues valid for this release.

Important known issues
----------------------------
These known issues are potential regressions or high impact issues.
SoftDevice Controller DRGN-15852 and DRGN-15852: Affecting scanning on the nRF53 Series.
HomeKit KRKNWK-10611: DFU will fail when using external flash memory for update image storage.
```

Reply:
This is done now.